commit a83b35e5d84881edec395ee20e12a4dc42bac33b (refs/changes/21/1121/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-14T14:16:00-08:00 (2 years, 10 months ago)
    
    TOOLS-1636 updates-imgadm et al with node v4 will not flush stdout on exit

diff --git a/package.json b/package.json
index 69bd22e..f72a187 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "dependencies": {
     "manta": "3.1.3",
     "smartdc": "7.3.0",
-    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.3"
+    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#38332b1"
   },
   "engines": {
     "node": ">=0.8"
